contributing/getting-started-e2e.md

```diff
@@ -184,13 +184,20 @@ Running the `gulp` command will compile the code and start up a Node.js server l
 
 You can browse the [http://localhost:8000/examples](http://localhost:8000/examples) directory to see some demo pages for various AMP components and combination of components.
 
-Note that by default each of the pages in the /examples directory actually uses the production version of AMP JavaScript; you can verify this by loading an example page in your browser, viewing the source and seeing that it is loading the AMP JavaScript from cdn.ampproject.org.
+Note that by default each of the pages in the /examples directory actually uses the unminified AMP JavaScript from your local server; you can verify this by loading an example page in your browser, viewing the source and seeing that it is loading the unminified AMP JavaScript from `http://localhost:8000`.
```

Comment:
I would say no need for the verifying part since serving unminified js from local server is indeed the desired behavior. 

what about:
`Note that by default each of the pages in the /examples directory uses the unminified AMP JavaScript from your local server. You can also change which JS to load from local server by hitting the `/serve_mode=<mode>` endpoint:`